v1/operator_test.go

```diff
@@ -0,0 +1,99 @@
+package v1
+
+import (
+	"testing"
+
+	client "github.com/hashicorp/nomad-openapi/clients/go/v1"
+	"github.com/hashicorp/nomad/command/agent"
+	"github.com/stretchr/testify/require"
+)
+
+// boolToPtr returns the pointer to a boolean
+func boolToPtr(b bool) *bool {
```

Comment:
we have these methods in `nomad/helpers` already. Before we merge, we should refactor to using those.

Reply:
Done.  Added `nomad/helper` import and refactored to call `helper.BoolToPtr` instead!